components/logo-carousel/logo-carousel.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {TweenLite, TimelineLite, TimelineMax}  from 'gsap';
+
+class LogoCarousel extends React.Component {
+	constructor() {
+		super();
+		this.animate = this.animate.bind(this);
+	}
+
+	componentDidMount() {
+		window.addEventListener('load', this.animate);
+	}
+
+	animate() {
+		var slider = document.querySelector('[data-marquee-slider]');
+		var sliderContent = slider.querySelector('[data-marquee-content]');
+		var sliderContentWidth = sliderContent.getBoundingClientRect().width;
+		var sliderSpeed = slider.dataset.marqueeSlider;
+
+		var carouselAnimation = new TimelineMax({repeat:-1});
+		carouselAnimation.add(TweenLite.to(slider, 1, {ease:Linear.easeNone, left:-sliderContentWidth}))
+						 .duration(sliderSpeed);
+	}
+
+	render() {
+		return(
+			<div className="logo-carousel container clearfix">
+				<h2 className="content">{ this.props.title }</h2>
+
+				<div className="marquee container-inner">
+					<div className="marquee__slider" data-marquee-slider={this.props.animationSpeed}>
```

Comment:
we don't use bem in this project, so you can just do: `marquee-slider`. This is also true for the other classes